<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CCMConfig</class>
 <widget class="QWidget" name="CCMConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>50</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CCM Configuration</string>
  </property>
  <layout class="QGridLayout" name="ccmconfig_lay">
   <item row="1" column="1">
    <widget class="QWidget" name="ccmParam" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="5" column="0" colspan="3">
       <widget class="QRadioButton" name="bottomRightMouse">
        <property name="text">
         <string>Capture mouse click as bottom-right</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QSpinBox" name="top">
        <property name="maximum">
         <number>65535</number>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QSpinBox" name="bottom">
        <property name="maximum">
         <number>65535</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="3">
       <widget class="QRadioButton" name="noMouse">
        <property name="text">
         <string>No mouse capture</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QRadioButton" name="sizeMouse">
        <property name="text">
         <string>Adjust patch size from mouse drag</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Patch size</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="patchWidth">
        <property name="maximum">
         <number>65535</number>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QSpinBox" name="patchHeight">
        <property name="maximum">
         <number>65535</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="3">
       <widget class="QRadioButton" name="topleftMouse">
        <property name="text">
         <string>Capture mouse click as top-left</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Top-left</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSpinBox" name="left">
        <property name="maximum">
         <number>65535</number>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Bottom-right</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QSpinBox" name="right">
        <property name="maximum">
         <number>65535</number>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Patch margin</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QSpinBox" name="hMargin">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="maximum">
         <number>65535</number>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QSpinBox" name="vMargin">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="maximum">
         <number>65535</number>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QPushButton" name="update_btn">
        <property name="text">
         <string>Update Grid</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="file_grp">
     <property name="title">
      <string>Calibration file</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="filegrp_lay">
      <item row="0" column="0">
       <widget class="QLabel" name="inputFile_lbl">
        <property name="text">
         <string>Input file</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="temperature">
        <property name="maximum">
         <number>100000</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="temperature_lbl">
        <property name="text">
         <string>Temperature</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QtExtra::FileWidget" name="inputFile" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="imgview_lay"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtExtra::FileWidget</class>
   <extends>QWidget</extends>
   <header location="global">QtExtra/filewidget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>temperature</tabstop>
  <tabstop>noMouse</tabstop>
  <tabstop>sizeMouse</tabstop>
  <tabstop>patchWidth</tabstop>
  <tabstop>patchHeight</tabstop>
  <tabstop>topleftMouse</tabstop>
  <tabstop>left</tabstop>
  <tabstop>top</tabstop>
  <tabstop>bottomRightMouse</tabstop>
  <tabstop>right</tabstop>
  <tabstop>bottom</tabstop>
  <tabstop>hMargin</tabstop>
  <tabstop>vMargin</tabstop>
  <tabstop>update_btn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
